playback/Cargo.toml

```diff
@@ -0,0 +1,29 @@
+[package]
+name = "librespot-playback"
+version = "0.1.0"
+authors = ["Sasha Hilton <sashahilton00@gmail.com>"]
+
+[dependencies.librespot-audio]
+path = "../audio"
+[dependencies.librespot-core]
+path = "../core"
+[dependencies.librespot-metadata]
+path = "../metadata"
+
+[dependencies]
+futures = "0.1.8"
+log = "0.3.5"
+
+alsa            = { git = "https://github.com/plietar/rust-alsa", optional = true }
+portaudio-rs    = { version = "0.3.0", optional = true }
+libpulse-sys    = { version = "0.0.0", optional = true }
+jack            = { version = "0.5.3", optional = true }
+libc            = { version = "0.2", optional = true }
+
+[features]
+alsa-backend = ["alsa"]
+portaudio-backend = ["portaudio-rs"]
+pulseaudio-backend = ["libpulse-sys", "libc"]
+jackaudio-backend = ["jack"]
+
+default = ["portaudio-backend"]
```

Comment:
This is causing the cross-compilation to fail.

We don't build any backends on Travis when cross-compiling, but `--no-default-features` does not propagate to dependencies.

Having `default = ["librespot-playback/portaudio-backend"]` in the root `Cargo.toml` is enough.

Reply:
Good to know, thanks for pointing that out. Will let Travis run and merge if successful.